script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -76,22 +83,32 @@ I ask Hisui in a whisper, trying not to be heard by Akiha.
 [sha:f7a3e64b23b5393ec51f82382367202f0b90bd8b]{
 -- Page 330, Offset 14732.
 -- 「……申し訳ありません。私には分かりかねます。……あくまで憶測ですが、秋葉さまはお疲れなのではないでしょうか」
-"...My deepest apologies. It is difficult for me to say... This is purely speculation, but perhaps it could be the case that Akiha-sama is simply fatigued."
+"...My deepest apologies. It is difficult for me to say for sure...
+ This is purely speculation, but perhaps it is the case that Akiha-sama is simply fatigued."
 }
 [sha:bb4629d224a45f278092ae96d6f5b452e4a47018]{
 -- Page 330, Offset 14733.
 -- 「……そっか。そりゃそうだよな。いつも不機嫌だけど、今はとくにご機嫌ななめだし。ほんと、ご苦労さまだこと」
-"...I see. I guess you're right. She's usually in a bad mood, but today it's particularly bad. I really have to applaud her hard work."
+// Initially thought that ご苦労さまだこと was sarcastic (wow, she's put so much effort into being a bastard)
+// but with the next line and the VA it sounds a bit more like he's commiserating with her.
+"...I see. I guess you're right.
+ She's always a little sullen, but this right here is a new level of ill temper.
+ She must really have been run off her feet all afternoon."
```

Comment:
run her feet off

Reply:
Run off their feet is the phrase I was thinking of here
https://idioms.thefreedictionary.com/run+off+my+feet

Reply:
I have literally never heard this idiom

Reply:
Hrm, if it's not that common maybe I'd better go for something more obvious